traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -496,7 +556,6 @@ func createV40(w http.ResponseWriter, r *http.Request, inf *api.APIInfo, dsV40 t
 		return nil, http.StatusInternalServerError, nil, errors.New("error writing to audit log: " + err.Error())
 	}
 
-	dsV40 = tc.DeliveryServiceV40(ds)
```

Comment:
I think we actually need this line still, because `ds` has been mutated throughout this function, and we want to return those mutations in the response.